app/controllers/users/sessions_controller.rb

```diff
@@ -74,10 +74,20 @@ def increment_session_bad_password_count
     def process_locked_out_session
       warden.logout(:user)
       warden.lock!
-      flash[:error] = t('errors.sign_in.bad_password_limit')
+
+      flash[:error] = t(
+        'errors.sign_in.bad_password_limit',
+        time_left: locked_out_time_remaining,
+      )
       redirect_to root_url
     end
 
+    def locked_out_time_remaining
+      locked_at = session[:max_bad_passwords_at]
+      window = IdentityConfig.store.max_bad_passwords_window_in_seconds.seconds
+      distance_of_time_in_words(Time.zone.now, (locked_at + window), true, except: :seconds)
```

Comment:
I can see how `except: :seconds` would make sense for a time period longer than a minute, but with a time period shorter than a minute (e.g. the screenshot in your PR comment), I might personally expect it to be shown as "30 seconds" rather than as a "less than a minute".

Reply:
okay I was just worried it would get too in the weeds.

Reply:
Yeah, that's fair. Maybe we could drop the `except: :seconds` and just let it always show the seconds? I think we do this elsewhere, e.g. "try again in 5 minutes and 26 seconds". (even if it doesn't necessarily countdown automatically)